geoprocessing/src/utils/file.utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * File Utils
+ */
+
+import { MulterOptions } from '@nestjs/platform-express/multer/interfaces/multer-options.interface';
+import { createReadStream, readdirSync } from 'fs';
+
+/**
+ * ES5 import to avoid TS complaining
+ */
+const multer = require('multer');
```

Comment:
https://www.npmjs.com/package/@types/multer

Reply:
@kgajowy @types/multer are already present as dev dependecies tho
How can I solve this?

Reply:
```

import { diskStorage } from 'multer';

/**
 * Options for Multer
 */
export const uploadOptions: MulterOptions = {
  storage: diskStorage({
    destination: function (
      req,
      file,
      cb: (error: Error | null, destination: string) => void,
    ) {
      cb(null, '/../tmp/');
    },
    filename: (req, file, cb: (error: Error | null, _: string) => void) =>
      cb(null, Date.now() + '-' + file.originalname),
  }),
};

```

Reply:
![image](https://user-images.githubusercontent.com/3466388/117635242-ecfe8c00-b17f-11eb-82e6-aeae9cceb925.png)